tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -2116,11 +2116,15 @@ <h2>Show similarity to selected datapoint</h2>
     const inferenceSquaredErrorStr = ' Inference squared error';
     const inferenceScoreStr = ' Inference score';
     const exampleIdKey = ' Datapoint ID';
-    const classificationResultsKey = 'probabilities';
-    const regressionResultsKey = 'score';
-    const attributionPrefix = 'pig_';
-    const attributionSparseKeyPrefix = 'pig_keys_';
-    const attributionSparseValuePrefix = 'pig_vals_';
+
+    // Special feature names and feature name prefixes and suffixes for parsing
+    // inference and attribution information from TFMA-generated input.
+    const classificationResultsKey = 'predictions__probabilities';
```

Comment:
no action required: though this achieve consistency within file, constants should be ALL_CAPS